components/AddContributorsHeader/add-contributors-header.tsx

```diff
@@ -42,7 +44,9 @@ const AddContributorsHeader = ({
         <h1 className="self-start text-2xl flex items-center">
           <Link
             className="inline-block p-3 mr-2 border rounded-lg cursor-pointer bg-light-slate-1"
-            href={`/lists/${list.id}/edit`}
+            href={
+              workspaceId ? `/workspaces/${workspaceId}/contributor-insights/${list.id}/edit` : `/lists/${list.id}/edit`
+            }
```

Comment:
This redirect should work 👍🏼 with personal workspaces/other workspaces: in the end, they're all just workspaces. More details here: https://github.com/open-sauced/api/pull/558